prod/westeurope/support/resource_group/terragrunt.hcl

```diff
@@ -0,0 +1,12 @@
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_resource_group?ref=v2.1.5"
+}
+
+inputs = {
+  name = "support"
```

Comment:
backoffice?

Reply:
I didn't want to rename the whole resource group... as it's the equivalent to destroy it and create a new one.